default.nix

```diff
@@ -1,16 +1,16 @@
-with (import <nixpkgs> {});
-rec {
-  polkadot-launch = mkYarnPackage {
-    name = "polkadot-launch";
-    src = ./.;
-    buildPhase = ''
-      yarn build
-    '';
-    postInstall = ''
-      chmod +x $out/bin/polkadot-launch
-    '';
-    packageJSON = ./package.json;
-    yarnLock = ./yarn.lock;
-    yarnNix = ./yarn.nix;
-  };
+{ pkgs ? import <nixpkgs> { } }:
+pkgs.mkYarnPackage {
+  name = "polkadot-launch";
+  src = builtins.filterSource
+    (path: type: type != "directory" || baseNameOf path != "bin")
+    ./.;
```

Comment:
Lol. Next level shit is going on 😂😂

Reply:
i find this syntax offensive

Reply:
At least it is deterministically offensive.